source/fundamentals/crud/run-command.txt

```diff
@@ -63,7 +67,10 @@ Information <addl-info-runcommand>`.
 Response
 --------
 
-Each method returns a document or a cursor containing documents with the
+Each method returns a result or a cursor that contains the response
+from the database after you run a command. Each database command
+requests different information, so the response content
```

Comment:
I: Is every command just a query (requesting info from the DB) or are there other types too?

S: If the commands do very different things, maybe change to "Each database command performs a different function" or similar

Reply:
[Some commands](https://www.mongodb.com/docs/manual/reference/command/nav-crud/) do make changes to the database/collection. I will change the language accordingly.